### DISCUSSION TITLE
How to toggle the file explorer visibility ?

### DISCUSSION THREAD

**Discussion by tito** [zed-industries/zed#20627] [Feature Requests]
I checked the documentation, and search on the issues, but didn't find a way to hide or show the file explorer.
It's not always necessary, and would like to hide it most of the time. Is there a command for that ?

**Comment by tito** [comment]
Ok, it look like i didn't search enough. It was just on the UI itself and the name for this is not sidebar, pane, or such... but dock.
On linux using Zed 0.162.0, Control+B never worked to close the pane (i was used to this in nvim), the editor cursor actually goes to the end of the file.

Adding this into the keymap.json make it work for me:

```json
  {
    "context": "Workspace",
    "bindings": {
      "ctrl-b": "workspace::ToggleLeftDock"
    }
  },
  {
    "context": "Editor",
    "bindings": {
      "ctrl-b": "workspace::ToggleLeftDock"
    }
  },
```

**Reply by dawoodjee** [reply]
Found out its `Command + R` on MacOS
